civicase.php

```diff
@@ -637,6 +622,8 @@ function civicase_civicrm_entityTypes(&$entityTypes) {
     'class' => 'CRM_Civicase_DAO_CaseContactLock',
     'table' => 'civicase_contactlock',
   );
+
+  _civicase_update_case_type_entity($entityTypes);
```

Comment:
Why don't we give this a more descriptive name e.g `_civicase_add_case_category_case_type_entity`